PythonRegression/ciglue.sh

```diff
@@ -6,31 +6,30 @@ DOCKER_REGISTRY=dyrellc/iri-network-tests
 ERROR=0
 
 git clone --depth 1 --branch yaml_configuration https://github.com/karimodm/iri-network-tests.git iri-network-tests
+
 cd iri-network-tests
 git pull
+echo "iri-network-tests revision: "; git rev-parse HEAD
+/iri-regression into glue
 virtualenv venv
 source venv/bin/activate
 pip install -r requirements.txt
-
 cd ..
+
 for machine_dir in tests/features/machine?; do
   python iri-network-tests/create_cluster.py -u $DOCKER_REGISTRY -d -c $machine_dir/config.yml -o $machine_dir/output.yml
   if [ $? -ne 0 ]; then
     ERROR=1
     python <<EOF
-from __future__ import print_function
```

Comment:
@karimodm 

Note that aloe tests will all run on python 3. I think your scripts should also be on 3.